src/main/java/frc/robot/subsystems/led/Led2023.java

```diff
@@ -1,50 +1,60 @@
 package frc.robot.subsystems.led;
 
+import edu.wpi.first.math.util.Units;
 import edu.wpi.first.wpilibj.DriverStation;
 import edu.wpi.first.wpilibj.RobotController;
 import edu.wpi.first.wpilibj.Timer;
 import edu.wpi.first.wpilibj.util.Color;
-import edu.wpi.first.wpilibj.util.Color8Bit;
+import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.lib.leds.DoubleLEDStrip;
 import frc.lib.leds.LEDManager;
 import frc.robot.RobotConstants;
 import frc.robot.commands.arm.ArmCalibrateCMD;
+import frc.robot.commands.arm.ArmCalibrateZeroAtHomeCMD;
 import frc.robot.commands.arm.ArmFloorCMD;
 import frc.robot.commands.arm.ArmScoreHighNodeCMD;
 import frc.robot.commands.arm.ArmScoreMidNodeCMD;
 import frc.robot.commands.arm.ArmShelfCMD;
+import frc.robot.commands.auto.complex.OnlyBackup;
+import frc.robot.commands.auto.complex.OnlyScore;
+import frc.robot.commands.auto.complex.ScoreAndBackUp;
 import frc.robot.commands.intakerelease.HoldCMD;
 import frc.robot.commands.intakerelease.IntakeAndRaise;
 import frc.robot.commands.intakerelease.IntakeCMD;
 import frc.robot.commands.intakerelease.ReleaseCMD;
 import frc.robot.subsystems.arm.Arm;
+import frc.robot.subsystems.drive.Drive;
 import frc.robot.subsystems.intakerelease.IntakeRelease;
+import org.littletonrobotics.junction.networktables.LoggedDashboardChooser;
 
 public class Led2023 extends SubsystemBase {
   public DoubleLEDStrip ledStrip;
-  private final boolean USE_BATTERY_CHECK = true;
-  private final double BATTER_MIN_VOLTAGE = 9.0; // 10.0
-  private final boolean CHECK_ARM_CALIBRATION = true;
+
+  private Timer balanceTimer = new Timer();
+  private Timer defaultTimer = new Timer();
+  private ColorScheme lastColorScheme;
+  private boolean doneBalancing = false;
   private boolean FINISHED_RAINBOW_ONCE = false;
```

Comment:
CAPS_CASE should be only used for constants. This should be finishedRainbowOnce.